include/bluetooth/services/ots_srv.h

```diff
@@ -0,0 +1,644 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTS_SRV_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTS_SRV_H_
```

Comment:
Btw, the S on OTS already means service so we could possible drop SRV term use on the APIs, etc.

Reply:
SRV is for server, as opposed to OTS_CLI for client.

Reply:
Profiles are the ones that implement client side procedures, Services are always server side procedures, lets please not start mixing these up.

Reply:
I guess the terminology is in fact that we have a Object Server that has one or more Object Transfer Services (defined by the OTS spec), and on the other side we have the Object Client that implements the procedures defined by the Object Transfer Profile. In reality service != server, and profile != client. 

We could rename things to be `OT_SRV` and `OT_CLI` or even `OBJ_SRV` and `OBJ_CLI` if we want to use the names by the OTP spec: 

![Screenshot_2020-09-02_09-12-38](https://user-images.githubusercontent.com/2520413/91943263-99889400-ecfc-11ea-91f9-8c28425ad840.png)

Reply:
Thanks @Thalley. I agree that there does not seem to be a mapping from profile to client.
Also I would prefer to use the _cli_ and _srv_ to separate the client and server APIs to avoid confusion, and this pattern is already used in the Mesh models, which are similar to GATT services.

Reply:
This is resolved as part of a broader discussion on Slack